harness/determined/cli/workspace.py

```diff
@@ -22,6 +22,29 @@
     "Agent Group",
 ]
 
+workspace_arg: Arg = Arg("--workspace-name", type=str, help="workspace name")
+
+
+def get_workspace_names(session: api.Session) -> Dict[int, str]:
```

Comment:
we might decide to do this translation in master at the api level which would move these checks there.